modules/simple-payments/simple-payments.php

```diff
@@ -94,9 +95,11 @@ function output_shortcode( $data ) {
 <div class='{$data[ 'class' ]} jetpack-simple-payments-wrapper'>
 	<div class='jetpack-simple-payments-title'>{$data['title']}</div>
 	<div class='jetpack-simple-payments-description'>{$data['description']}</div>
-	<div class='jetpack-simple-payments-price'>{$data['price']}</div>
-	{$items}
-	<div class='jetpack-simple-payments-button' id='{$data['dom_id']}_button'></div>
+	<div class='jetpack-simple-payments-payment-row'>
```

Comment:
We should avoid using underscores for front-end staff. https://make.wordpress.org/core/handbook/best-practices/coding-standards/css/#selectors